### PR TITLE
Address sphinx warnings when building docs

### PR DESCRIPTION
**Pull request by rahulporuri**
Addresses most of the warnings reported in #563 

This PR addresses most of the sphinx documentation warnings reported when building the documentation.

### CODE REVIEW

**Comment by kitchoi** [comment]
These two files look orthogonal:
pyface/image/library/icons.zip 
pyface/image/library/std.zip 

Could you revert the changes to these two files please? I still haven't got round to understand why they keep being changed.

File: pyface/timer/do_later.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def do_later(callable, *args, **kwargs):
     ----------
     callable : callable
         The callable to call in 50ms time.
-    *args, **kwargs :
+    args, kwargs : tuple, dict
```
Comment (kitchoi) [inline pyface/timer/do_later.py:41]: Would this suggest `args` and `kwargs` can be passed as positional arguments?

Comment (rahulporuri) [reply pyface/timer/do_later.py]: reverted the change to use `*args` and `**kwargs` again.

**Review by kitchoi — APPROVED** [review]
LGTM

**Comment by kitchoi** [comment]
Build on OSX fails due to the sporadic error noted in #556.

**Comment by corranwebster** [comment]
> Could you revert the changes to these two files please? I still haven't got round to understand why they keep being changed.

They are automatically regenerated based on timestamps (and a new mtime means git wants to update them) in `pyface.image.image`.  I remember fixing a timezone problem in them a few years ago after a plane flight broke images in Pyface (so they now use UTC at least), but it's quite possible that they have other issues around the regeneration of the files.  In particular, if you unzip them I am pretty sure that it will re-zip them at least once which will reset the mtime, but there may be other code which is in play.

**Pull request merged**